python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -895,6 +895,13 @@ def updateTableColumn(self, table, column, new_name=None, data_type=None, not_nu
                     self.quoteString(new_name), schema_where, self.quoteString(tablename), self.quoteString(column))
                 self._execute(c, sql)
 
+        # comment the column
+        if comment is not None:
+            # Add the comment on the field
+            schema, tablename = self.getSchemaTableName(table)
+            sql = u"COMMENT ON COLUMN %s.%s.%s is '%s'" % (schema, tablename, column, comment)
```

Comment:
do we need `u` prefix in Python 3?

Reply:
`u` is no longer required (and should therefore be omitted).
I think it's also worth to quote the schame, table and column names with `"`